pyproject.toml

```diff
@@ -10,12 +10,13 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.6, <4.0"
 Flask = "^1.1.2"
 authutils = "^5.0.5"
 boto3 = "^1.17.48"
 cryptography = "2.8"
 requests = "^2.25.1"
+importlib-metadata = "^4.8.1"
```

Comment:
why do we need to add importlib-metadata?

Reply:
poetry install and poetry lock works fine. However when I try to build with docker I get a 
`ModuleNotFoundError: No module named 'importlib_metadata` error. So I added this to get the build to run.